services/tacoService.js

```diff
@@ -0,0 +1,32 @@
+const Taco = require('../models/Taco');
+
+const createTaco = (recipientUserId, giverUserId, number, reason) => {
+    let taco = new Taco();
+    taco.id = taco._id;
+    taco.recipientUserId = recipientUserId;
+    taco.giverUserId = giverUserId;
+    taco.number = number;
+    taco.reason = reason;
+    taco.save((err) => {
+        if(err) { console.error(err)}
+    });
+};
+
+const getNumberOfGiftedTacosByUserInLastDay = (userid) => {
+    return Taco.find({
+        giverUserId: userId,
+        createdAt: { $gt: new Date(new Date().setHours(0,0,0,0)) }
+    });
+};
+
+const getTacosForUserAllTime = (userId) => {
```

Comment:
Unused function, and this value should be stored in the scoreboard anyways

Reply:
Yeah I figured this could be useful in the future if people can redeem tacos

Reply:
Wouldn't the total tacos just get deducted from the scoreboard? Maybe add a `tacoBalance` to the scoreboard